app/assets/javascripts/Measure.js

```diff
@@ -2,46 +2,49 @@ const mongoose = require('mongoose');
 const Code = require('./basetypes/Code');
 const Interval = require('./basetypes/Interval');
 const Quantity = require('./basetypes/Quantity');
+const CQLStatementDependencySchema = require('./CQLStatementDependency');
+const CQLLibrarySchema = require('./CQLLibrary');
 
-const [mNumber, mString, mBoolean, Mixed, ObjectId, mDate] = [
-  mongoose.Schema.Types.Number,
-  mongoose.Schema.Types.String,
-  mongoose.Schema.Types.Boolean,
+const [Mixed, ObjectId, mDate] = [
   mongoose.Schema.Types.Mixed,
   mongoose.Schema.Types.ObjectId,
   mongoose.Schema.Types.Date,
 ];
 
 const MeasureSchema = mongoose.Schema(
   {
-    // ID/other measure information
-    id: mString,
-    measure_id: mString,
```

Comment:
measure_id is used in "as_hqmf_model"

Reply:
Good catch. It seems like `measure_id` is always equivalent to `hqmf_id` when set; are you ok with us using `hqmf_id` in place of `measure_id` in the `as_hqmf_model` method?